src/libs/actions/Report.js

```diff
@@ -516,6 +526,13 @@ function updateReportWithNewAction(reportID, reportAction) {
     if (lodashGet(reportAction, 'actorEmail') === 'concierge@expensify.com') {
         return;
     }
+
+    // When a new message comes in, if the New marker is not already set (newMarkerSequenceNumber === 0) and, set the
+    // marker above the incoming message.
+    if (lodashGet(allReports, reportID).newMarkerSequenceNumber === 0 && updatedReportObject.unreadActionCount > 0) {
```

Comment:
It's very possible that this report does not yet exist. And if `lodashGet(allReports, reportID)` is `undefined` then we will try and fail to access `newMarkerSequenceNumber`.

In the case that it doesn't exist I think we would treat that as the same as having a `newMarkerSequenceNumber === 0` so we should do this instead...

```suggestion
    if (lodashGet(allReports, [reportID, 'newMarkerSequenceNumber'], 0) === 0 && updatedReportObject.unreadActionCount > 0) {
```

Reply:
Ah! That makes sense. Good catch!